config/puma.rb

```diff
@@ -9,7 +9,7 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch("PORT") { 3000 }
+port        ENV.fetch("PORT") { 3001 }
```

Comment:
3001 is being used by the Citizen app, I'd suggest 3002 instead.

Reply:
I had to change this locally while development due to the provider frontend and API still being stuck on the same port. Afterwards, a local port hierarchy was created